libcontainer/standard_init_linux.go

```diff
@@ -156,9 +150,24 @@ func (l *linuxStandardInit) Init() error {
 	// do this before dropping capabilities; otherwise do it as late as possible
 	// just before execve so as few syscalls take place after it as possible.
 	if l.config.Config.Seccomp != nil && !l.config.NoNewPrivileges {
-		if err := seccomp.InitSeccomp(l.config.Config.Seccomp); err != nil {
+		seccompFd, err := seccomp.InitSeccomp(l.config.Config.Seccomp)
+		if err != nil {
 			return err
 		}
+		if seccompFd != -1 {
+			err := syncParentSeccomp(l.pipe, seccompFd)
+			if err != nil {
+				return errors.Wrap(err, "sync parent seccomp")
+			}
+			err = unix.Close(seccompFd)
+			if err != nil {
+				return errors.Wrap(err, "close seccomp fd")
+			}
+		}
```

Comment:
Maybe move this part out to a function, as this code is used twice.

Reply:
Fixed, thanks. Moved to `syncParentSeccomp()`